config/checkstyle_checks.xml

```diff
@@ -350,6 +350,13 @@
     <module name="SuppressionXpathFilter">
       <property name="file" value="${checkstyle.suppressions-xpath.file}"/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+      <property name="files" value="IGNORETHIS"/>
+      <property name="checks" value="IGNORETHIS"/>
+      <property name="message" value="IGNORETHIS"/>
+      <property name="id" value="IGNORETHIS"/>
+      <property name="query" value="IGNORETHIS"/>
+    </module>
```

Comment:
I discussed with @rnveach , how to use this filter, he is thing now , ones he agree, he will share details.

Reply:
@rnveach

Reply:
@rnveach , please share your thoughts on model of Filter usage.